CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Release notes
 
+## v0.126
+
+#### Feat
+
+- Similarly to Vue, inside component HTML, you can now shorten `<div></div>` with self-closing tags `<tag />`.
+
+    The shortened version is expanded at render time. So this:
+
+    ```html
+    <div style="background-image: url('...');" />
+    ```
+
+    will be expanded to:
+
+    ```html
+    <div style="background-image: url('...');">
+    </div>
+    ```
```

Comment:
This feels very random, don't think it has anything to do with django-components at all.

Reply:
Oki, I'll move this out of this PR, and instead leave it for django-vue.

Reply:
That is a good solution imho! ⭐